app/services/checksum_validator.rb

```diff
@@ -1,7 +1,13 @@
+require 'moab_validation_handler.rb'
+
 # code for validating Moab checksums
 class ChecksumValidator
+  include ::MoabValidationHandler
+
+  attr_reader :results, :endpoint, :full_druid, :preserved_copy, :bare_druid
 
-  attr_reader :checksum_results, :endpoint, :full_druid, :preserved_copy, :bare_druid
+  alias druid bare_druid
```

Comment:
I didn't know about alias, so I did some reading on it and see that some folks advocate for `alias_method` over `alias`. For ex. (https://blog.bigbinary.com/2012/01/08/alias-vs-alias-method.html). Do you think there's any value in switching over?

Reply:
I don't have strong feelings either way.  Lowest effort - leave it as it is?

Reply:
i went and implemented this suggestion, switching over to `alias_method`, but then rubocop complained that i should prefer alias in a class body (i guess as opposed to in a method call, as in the example link):
```
Style/Alias: Use alias instead of alias_method in a class body.
```

i'm not inclined to make a class method to do the aliasing as in the example in the above link, just because it seems a little convoluted.  though, since we're using inheritance of a sort here, the argument in that link is sorta convincing.

@atz has good decomposition and OO instincts and is more of a rubyist than me, so maybe he has an opinion?

in the interest of getting in what's hopefully an improvement, i'm inclined to ticket this discussion as follow-on work, unless joe weighs in with clear explanation as to why we should definitely go one way or the other.

Reply:
filed #686